dask/array/tests/test_rechunk.py

```diff
@@ -772,13 +772,6 @@ def test_old_to_new_large():
     assert result == expected
 
 
-def test_changing_raises():
-    with pytest.raises(ValueError) as record:
-        old_to_new(((np.nan, np.nan), (4, 4)), ((np.nan, np.nan, np.nan), (4, 4)))
-
-    assert "unchanging" in str(record.value)
```

Comment:
Is this test now no longer valid?

Reply:
Yes, the test is no longer valid. It triggered validation within `old_to_new`. but with this change, `old_to_new` assumes that validation has already taken place (see changes to docstring).

Reply:
I've added the same test case in a different place where it's still valid.

Reply:
Thanks, missed that subtlety.